resources/frontend_shared/static_viz_interface.js

```diff
@@ -47,11 +47,12 @@ function combo_chart(series, settings, colors) {
   });
 }
 
-function timeseries_waterfall(data, labels, settings) {
+function timeseries_waterfall(data, labels, settings, colors) {
```

Comment:
Do you think this should better name `instanceColor` as in
https://github.com/metabase/metabase/blob/5e9f03ae4b8bd40cae09ab375b5715d3b5b9ce82/frontend/src/metabase/static-viz/lib/colors.ts#L4

At glance, I might thought this is some kind of arbitrary color setting for this specific chart rather than an instance color.

Also, I think the `colors` type for `combo-chart` is out of date.
https://github.com/metabase/metabase/blob/5e9f03ae4b8bd40cae09ab375b5715d3b5b9ce82/frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx#L21

https://github.com/metabase/metabase/blob/5e9f03ae4b8bd40cae09ab375b5715d3b5b9ce82/frontend/src/metabase/static-viz/components/LineAreaBarChart/types.ts#L1-L6

I can clearly see that we pass instance colors to combo chart 

https://github.com/metabase/metabase/blob/5e9f03ae4b8bd40cae09ab375b5715d3b5b9ce82/src/metabase/pulse/render/body.clj#L318

`settings` is a value from calling `->ts-viz` which is then passed to `js-svg/combo-chart` on line 511

https://github.com/metabase/metabase/blob/5e9f03ae4b8bd40cae09ab375b5715d3b5b9ce82/src/metabase/pulse/render/body.clj#L506-L511

Reply:
Thanks, renamed that. Regarding combo-chart, it expects colors in a different way because that was before the instance colors settings revamp. I'll update it separately